config/vendor-api-v1.yml

```diff
@@ -1181,11 +1183,13 @@ components:
           type: string
           nullable: true
           description: The candidate’s description of their disability, if they selected “Other” and entered a value. This corresponds to HESA disability code `96`.
+          maxLength: 10240
```

Comment:
🤔 Maybe we should just leave those out anyway for now, until we limit them to reasonable values

Reply:
I think it's ok — assuming these are baggy enough for existing data?